lib/redux/middleware/sentry_api_middleware.dart

```diff
@@ -15,49 +16,43 @@ class SentryApiMiddleware extends MiddlewareClass<AppState> {
   @override
   dynamic call(Store<AppState> store, action, next) {
     if (action is FetchOrganizationsAndProjectsAction) {
+      store.dispatch(LoadingAction(true));
```

Comment:
We should rename `LoadingAction` to something like `OrgsAndProjectsLoadingAction`